aio/src/app/shared/analytics.service.ts

```diff
@@ -0,0 +1,93 @@
+import { Inject, Injectable } from '@angular/core';
+
+import { environment } from '../../environments/environment';
+import { WindowToken } from 'app/shared/window';
+import { unwrapScriptUrlForSink, scriptUrl } from 'safevalues';
```

Comment:
Nit: Order imported identifiers alphabetically 😇

```suggestion
import { scriptUrl, unwrapScriptUrlForSink} from 'safevalues';
```